test/src/androidTest/java/com/quickbirdstudios/test/FullSurveyTest.kt

```diff
@@ -125,15 +117,17 @@ internal class FullSurveyTest : PageTest {
                         Assert.assertTrue(questionResult.answer.toString() == NumberStepInput)
                     is MultipleChoiceQuestionResult -> {
                         val expectedResult = listOf(
-                            (activityRule
-                                .activity
-                                .multipleChoiceStep
-                                .answerFormat as MultipleChoiceAnswerFormat
+                            (
```

Comment:
no need for extra line.

Reply:
When i format it via klint it's seems like that.